src/sveltekit/sdk-setup.ts

```diff
@@ -50,6 +54,23 @@ export async function createOrMergeSvelteKitFiles(
   projectInfo: ProjectInfo,
   svelteConfig: PartialSvelteConfig,
 ): Promise<void> {
+  const selectedFeatures = await featureSelectionPrompt([
+    {
+      id: 'performance',
+      prompt: `Do you want to enable ${chalk.bold(
+        'Tracing',
+      )} to track the performance of your application?`,
+      enabledHint: 'recommended',
+    },
+    {
+      id: 'replay',
+      prompt: `Do you want to enable ${chalk.bold(
+        'Sentry Session Replay',
+      )} to get reproduction of frontend errors via user sessions?`,
```

Comment:
The sentence reads funky to me and theoretically, replay can also surface backend errors. What about something like this?

```suggestion
      )} to get a video-like reproduction of errors during a user session?`,
```

I think we should also adjust this in the other wizards where we use `featureSelectionPrompt`.